src/main/java/com/philips/swcoe/cerberus/cerebellum/codemetrics/java/results/CodeMetricsClassResult.java

```diff
@@ -7,53 +7,51 @@
-    private CodeMetricsResult responseForClass;
-    private CodeMetricsResult numberOfStaticInvocations;
-    private CodeMetricsResult linesOfCode;
-    private CodeMetricsResult returnQty;
-    private CodeMetricsResult loopQty;
-    private CodeMetricsResult comparisonsQty;
-    private CodeMetricsResult tryCatchQty;
-    private CodeMetricsResult parenthesizedExpsQty;
-    private CodeMetricsResult stringLiteralsQty;
-    private CodeMetricsResult numbersQty;
-    private CodeMetricsResult assignmentsQty;
-    private CodeMetricsResult mathOperationsQty;
-    private CodeMetricsResult variablesQty;
-    private CodeMetricsResult maxNestedBlocks;
-    private CodeMetricsResult anonymousClassesQty;
-    private CodeMetricsResult subClassesQty;
-    private CodeMetricsResult lambdasQty;
-    private CodeMetricsResult uniqueWordsQty;
-    private CodeMetricsResult numberOfMethods;
-    private CodeMetricsResult numberOfStaticMethods;
-    private CodeMetricsResult numberOfPublicMethods;
-    private CodeMetricsResult numberOfPrivateMethods;
-    private CodeMetricsResult numberOfProtectedMethods;
-    private CodeMetricsResult numberOfDefaultMethods;
-    private CodeMetricsResult numberOfAbstractMethods;
-    private CodeMetricsResult numberOfFinalMethods;
-    private CodeMetricsResult numberOfSynchronizedMethods;
-    private CodeMetricsResult numberOfFields;
-    private CodeMetricsResult numberOfStaticFields;
-    private CodeMetricsResult numberOfPublicFields;
-    private CodeMetricsResult numberOfPrivateFields;
-    private CodeMetricsResult numberOfProtectedFields;
-    private CodeMetricsResult numberOfDefaultFields;
-    private CodeMetricsResult numberOfFinalFields;
-    private CodeMetricsResult numberOfSynchronizedFields;
-    private CodeMetricsResult modifiers;
+    private CodeMetricsDiffResult depthInheritanceTree;
+    private CodeMetricsDiffResult weightMethodClass;
+    private CodeMetricsDiffResult couplingBetweenObjects;
+    private CodeMetricsDiffResult responseForClass;
+    private CodeMetricsDiffResult numberOfStaticInvocations;
+    private CodeMetricsDiffResult linesOfCode;
+    private CodeMetricsDiffResult returnQty;
+    private CodeMetricsDiffResult loopQty;
+    private CodeMetricsDiffResult comparisonsQty;
+    private CodeMetricsDiffResult tryCatchQty;
+    private CodeMetricsDiffResult parenthesizedExpsQty;
+    private CodeMetricsDiffResult stringLiteralsQty;
+    private CodeMetricsDiffResult numbersQty;
+    private CodeMetricsDiffResult assignmentsQty;
+    private CodeMetricsDiffResult mathOperationsQty;
+    private CodeMetricsDiffResult variablesQty;
+    private CodeMetricsDiffResult maxNestedBlocks;
+    private CodeMetricsDiffResult anonymousClassesQty;
+    private CodeMetricsDiffResult subClassesQty;
+    private CodeMetricsDiffResult lambdasQty;
+    private CodeMetricsDiffResult uniqueWordsQty;
+    private CodeMetricsDiffResult numberOfMethods;
```

Comment:
Consistent naming here (is either numberOf or Qty) will likely be better for readers of this code to know if these are all the same (counts of some metric) or that there is something special or different between numberOf and Qty.

Reply:
Thanks for identifying, this Anti pattern, You are correct and I made it consistant accross the metrics package, by ensuring that every Quantifying variable in the metrics is named as variableCount . Please check ..